app/DynamicsAdapter/DynamicsAdapter.Web/searchApi.openapi.json

```diff
@@ -83,7 +83,6 @@
           "description": "The last name of the subject."
         },
         "dateOfBirth": {
-          "type": "string",
```

Comment:
you can't change this definition, this has to be taken from the searchApi, otherwise our service is not compatible with the searchApi

Reply:
ok. do not change it.